packages/grid_client/src/modules/zos.ts

```diff
@@ -50,78 +66,177 @@ class Zos {
     return await twinDeploymentHandler.handle([twinDeployment]);
   }
 
+  /**
+   * Ping a node to check its availability.
+   *
+   * @param {PingNodeOptionsModel} options - The options for pinging the node.
+   * @returns {Promise<boolean>} - A boolean indicating if the node is reachable.
+   * @decorators
+   * - `@expose`: Exposes the method for external use.
+   * - `@validateInput`: Validates the input options.
+   */
   @expose
   @validateInput
   async pingNode(options: PingNodeOptionsModel): Promise<boolean> {
     const nodeTwinId = await this.capacity.getNodeTwinId(options.nodeId);
     return await this.rmb.request([nodeTwinId], "zos.system.version", "", 10, 1);
   }
 
+  /**
+   * Get deployment information based on the contract ID.
+   *
+   * @param {ZOSGetDeploymentModel} options - The options for getting deployment information.
+   * @returns {Promise<any>} - A promise that resolves with the deployment information.
+   * @decorators
+   * - `@expose`: Exposes the method for external use.
+   * - `@validateInput`: Validates the input options.
+   */
   @expose
   @validateInput
-  async getDeployment(options: ZOSGetDeploymentModel) {
+  async getDeployment(options: ZOSGetDeploymentModel): Promise<any> {
```

Comment:
it should return a `Deployment` interface which is in the zos types
```suggestion
  async getDeployment(options: ZOSGetDeploymentModel): Promise<Deployment> {
```